frontend/packages/dev-console/src/utils/shared-submit-utils.ts

```diff
@@ -10,6 +10,12 @@ export const annotations = {
 
 export const dryRunOpt = { queryParams: { dryRun: 'All' } };
 
+const isDeployImageFormData = (
+  formData: DeployImageFormData | GitImportFormData,
+): formData is DeployImageFormData => {
+  return (formData as DeployImageFormData).isi !== undefined;
```

Comment:
May be write this as:
```suggestion
  return !_.isEmpty((formData as DeployImageFormData).isi);

Reply:
or `'isi' in formData` or `formData.hasOwnProperty('isis')` ?